cli/src/pixl_cli/_database.py

```diff
@@ -85,6 +90,7 @@ def _get_image_and_check_exported(
         session.query(Image)
         .filter(
             Image.extract == extract,
+            Image.project_name == slugify(message.project_name),
```

Comment:
oh this is already in the `Extract` table, so will need to query through that, instead of adding it to the Image